tests/test_jit.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+"""
+Unit tests for exporting models via TorchScript (i.e. JIT compilation).
+"""
+
+import os
+import regex
+import unittest
+
+import parlai.utils.testing as testing_utils
+from parlai.agents.repeat_label.repeat_label import RepeatLabelAgent
+from parlai.core.agents import create_agent
+from parlai.core.params import ParlaiParser
+from parlai.core.worlds import create_task
+from parlai.utils.bpe import Gpt2BpeHelper
+
+
+@testing_utils.skipUnlessGPU
+@testing_utils.skipUnlessTorch17
+class TestJit(unittest.TestCase):
+    def test_token_splitter(self):
+        """
+        Test TorchScriptable code for splitting tokens against reference GPT-2 version.
+        """
+
+        from parlai.scripts.jit_export import JitExport, ScriptableGpt2BpeHelper
+
+        # Params
+        tasks = ['taskmaster2', 'convai2']
+        compiled_pattern = regex.compile(Gpt2BpeHelper.PATTERN)
+
+        with testing_utils.tempdir() as tmpdir:
+            datapath = tmpdir
+
+            for task in tasks:
+                opt = JitExport.setup_args().parse_kwargs(
+                    task=task, datatype='train:ordered', datapath=datapath
+                )
+                agent = RepeatLabelAgent(opt)
+                teacher = create_task(opt, agent).get_task_agent()
```

Comment:
```suggestion
                # TODO(roller): make a proper create_teacher helper
                teacher = create_task(opt, agent).get_task_agent()
```

Reply:
Yes, added this (not committing this change since I've changed the path)